acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/signing/SecpSigningAcceptanceTest.java

```diff
@@ -0,0 +1,178 @@
+import static org.web3j.crypto.Sign.publicKeyFromPrivate;
+import static org.web3j.crypto.Sign.signedMessageToKey;
+import static tech.pegasys.signers.secp256k1.MultiKeyTomlFileUtil.createAzureTomlFileAt;
+import static tech.pegasys.signers.secp256k1.MultiKeyTomlFileUtil.createFileBasedTomlFileAt;
+import static tech.pegasys.signers.secp256k1.MultiKeyTomlFileUtil.createHashicorpTomlFileAt;
+
+import tech.pegasys.eth2signer.dsl.signer.SignerConfigurationBuilder;
+import tech.pegasys.eth2signer.tests.AcceptanceTestBase;
+import tech.pegasys.signers.hashicorp.dsl.HashicorpNode;
+import tech.pegasys.signers.secp256k1.HashicorpSigningParams;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.SignatureException;
+
+import com.google.common.io.Resources;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.vertx.core.json.JsonObject;
+import org.apache.tuweni.bytes.Bytes;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariables;
+import org.junit.jupiter.api.io.TempDir;
+import org.web3j.crypto.Sign.SignatureData;
+
+public class SecpSigningAcceptanceTest extends AcceptanceTestBase {
+  private static final byte[] DATA_TO_SIGN = "42".getBytes(UTF_8);
+  private static final String PRIVATE_KEY =
+      "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63";
+
+  private static final String SIGN_ENDPOINT = "/signer/sign/{publicKey}";
+
+  static final String FILENAME = "fe3b557e8fb62b89f4916b721be55ceb828dbd73";
+
+  @Test
+  @EnabledIfEnvironmentVariables({
+    @EnabledIfEnvironmentVariable(named = "AZURE_CLIENT_ID", matches = ".*"),
+    @EnabledIfEnvironmentVariable(named = "AZURE_CLIENT_SECRET", matches = ".*"),
+    @EnabledIfEnvironmentVariable(named = "AZURE_KEY_VAULT_NAME", matches = ".*")
+  })
+  @EnabledIfEnvironmentVariable(named = "AZURE_KEY_VAULT_NAME", matches = ".*")
+  public void signDataWithKeyInAzure(@TempDir Path tomlDirectory) {
+    final String clientId = System.getenv("AZURE_CLIENT_ID");
+    final String clientSecret = System.getenv("AZURE_CLIENT_SECRET");
+    final String keyVaultName = System.getenv("AZURE_KEY_VAULT_NAME");
+    createAzureTomlFileAt(
+        tomlDirectory.resolve("arbitrary_prefix" + FILENAME + ".toml"),
+        clientId,
+        clientSecret,
+        keyVaultName);
+
+    final SignerConfigurationBuilder builder = new SignerConfigurationBuilder();
+    builder.withKeyStoreDirectory(tomlDirectory);
+    startSigner(builder.build());
+
+    final Response response =
+        given()
+            .baseUri(signer.getUrl())
+            .filter(getOpenApiValidationFilter())
+            .contentType(ContentType.JSON)
+            .pathParam("publicKey", FILENAME)
```

Comment:
continuing to previous comment. `pathParam("identifier")`

Reply:
done